### PR TITLE
fix(combobox): enable submit and add a story

### PR DESCRIPTION
**Pull request by cal-smith**

### CODE REVIEW

File: src/combobox/combobox.stories.ts [hunk 1]
```diff
@@ -99,31 +88,39 @@ storiesOf("Components|Combobox", module)
...
+			</ibm-combo-box>
+		`,
+		props: getOptions({
+			submit: function(event) {
+				// so the action still shows up in the "actions" panel
+				action("submit")(event);
+				if (event.value.content) {
+					this.items = [
+						...event.items,
+						Object.assign({}, event.value, { selected: true})
```
Comment (Donisius) [inline src/combobox/combobox.stories.ts]:
```suggestion
						Object.assign({}, event.value, { selected: true })
```

**Comment by JehanX** [comment]
@zvonimirfras @Donisius If everything looks good, can you please merge it? Thanks

**Comment by cal-smith** [comment]
@JehanX If you have a pressing requirement or delivery date for a PR or Issue, definitely give us a specific time frame so we can prioritize things - we have a ton of requests so not everything can be done immediately

**Comment by cal-smith** [comment]
That said, this PR is good to go, so ... Merging!

**Pull request merged**